.travis.yml

```diff
@@ -6,13 +6,13 @@ branches:
   only:
     - master
     - starter
-    - issue-9-travis-pytest
-    - issue-11-add-Black-formatter
+    - issue-8-command-line
```

Comment:
You don't need to add this to have Travis builds run on your branch. Just make a PR with the branch and then the tests will run in that PR.